src/docs/product/performance/web-vitals.mdx

```diff
@@ -0,0 +1,67 @@
+---
+title: "Web Vitals"
+sidebar_order: 6
+description: "Learn more about Web Vitals in Sentry and how they give you a better understanding of your application's health."
+---
+
+Frontend transactions can additionally contain web vital information. With this view, you'll be able to examine a summary of the web vitals.
```

Comment:
We'll want a link to https://web.dev/vitals/ somewhere in this paragraph.

Reply:
Added in 680c827